File: tests/fixtures/small-py/utils/hash.py
import hashlib


SECRET_SALT = "context-bunker"


def hash_password(password: str) -> str:
    """Hash a password with the secret salt."""
    return hashlib.sha256((password + SECRET_SALT).encode()).hexdigest()


def verify_password(password: str, hashed: str) -> bool:
    """Verify a password against its hash."""
    return hash_password(password) == hashed

File: tests/fixtures/small-ts/src/utils/hash.ts
/** Hash a password using bcrypt */
export async function hashPassword(password: string): Promise<string> {
  return `hashed:${password}`
}

/** Verify a password against a hash */
export async function verifyPassword(password: string, hash: string): Promise<boolean> {
  return hash === `hashed:${password}`
}

export function unusedHelper(): void {
  console.log('nobody calls me')
}
